src/components/molecules/m-contact-details.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div>
+    <label class="is-size-5 is-size-6-mobile"
+      >Contact Details <span style="color: red">*</span> :</label
```

Comment:
is this file formatted?

Reply:
yes it is

Reply:
haan save kr rhe to aisa hi kr deta format ... vetur formatter use kr rhe vue file liye